### DISCUSSION TITLE
Manage the app from different users

### DISCUSSION THREAD

**Discussion by erakastrati** [Budibase/budibase#2507] [Help] [answered]
Hi there to all of you,

We are a small company, and I found this excellent no code low code platform that will definitely help us during our journey.
We are around 10 workers which will use the app that I created, mainly for the reason that it has a section where everyone can see its own tasks (using the current user filter).

The problem is that after X worker enters the budibase app, the Y workers can't enter into the app because the system itself puts a lock when X logged in. 
Now the only option is: Y calls X and asks him to remove the lock, so Y can enter the app to view its own tasks, which is not the best solution possible.

Is there a way we can get rid of this? How can two people be logged in into the app at the same app? Or at least would that be a plan in the near future?

Thanks a lot,
Era.

**Comment by joebudi** [comment]
Hey @erakastrati 

Thank you for your kind words. This is not possible right now but we do have plans to allow multiple people to edit an app at the same time. Out of curiosity, how would you expect this to work?

Cheers,
Joe

**Reply by erakastrati** [reply]
Actually I don't have any idea, what I think is after X logs into the app, why the system needs to lock that app, if it doesn't lock the app it would be the best way. Due to the reason that nothing would change, for ex. the method to log in, which we are using now, would be definitely the same.
I think that would be a simpler way of logging in.

**Reply by mjashanks** [reply]
Hey @erakastrati  - Just to be clear - its single user in **edit** / **developer** mode. Multiple users can of course log into the app that has been built and published.

Multi-user edit mode is difficult for us. We will allow it at some point, but not for a few months.

**Comment by shogunpurple** [comment, marked as answer]
@erakastrati just to clarify - you do mean 10 people developing/editing the app at once? Or just using it? If it's just using it, then you can publish your app and you and your co-workers can use it.

https://docs.budibase.com/publish/introduction-to-deploy

**Comment by erakastrati** [comment]
Hi to all of you, 
I meant in the way that we are two admins(which is not a big problem after the system is locked), and I just saw there is an option to view a published app in the user mode, which is what I wanted to achieve. The problem is solved now.
Thanks for the responses to all of you, keep rocking with this great work.
